SPECS/ksh/ksh.spec

```
@@ -80,12 +74,10 @@ touch %{buildroot}%{_bindir}/rksh
 touch %{buildroot}%{_mandir}/man1/rksh.1.gz
 
 %check
-# We run more tests as non-root user
-chmod g+w . -R
 useradd test -G root -m
-
-# Disabling tests as they tend to freez and the package is low-pri at the moment.
-false && sudo -u test ./bin/shtests --compile
+# disable pty tests as they tend to freeze
+find src/cmd/ksh93/tests -name "pty.sh" -delete
+bin/package test
```

This is looking nice. But can we still run the tests as non-root user? I believe that allowed more tests to run.
```suggestion
sudo -u test bin/package test
```

If that's no longer the case, please ignore this comment.;)